runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -426,6 +439,7 @@ public static class CorfuRuntimeParametersBuilder extends RuntimeParametersBuild
             private int streamBatchSize = 10;
             private int checkpointReadBatchSize = 5;
             private Duration runtimeGCPeriod = Duration.ofMinutes(20);
+            private Duration mvoAutoSyncPeriod = Duration.ofMinutes(5);
```

Comment:
It doesnt seem like this config is being used.

Reply:
Removed. This was a left over from previous code.